promu.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: promu
+  version: 0.17.0
+  epoch: 0
+  description: Prometheus Utility Tool
+  copyright:
+    - license: Apache-2.0
+
+update:
+  enabled: true
+  github:
+    identifier: prometheus/promu
+    strip-prefix: v
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/prometheus/promu.git
+      tag: v${{package.version}}
+      expected-commit: 3912dec4ab83971903015cc7b2a8d8ff93b73910
+
+  - uses: go/bump
+    with:
+      deps: |-
+        golang.org/x/oauth2@v0.27.0
+
+  - uses: go/build
+    with:
+      output: promu
+      packages: ./main.go
+
+test:
+  environment:
+    contents:
+      packages:
+        - go
+        - yq
+  pipeline:
+    - name: Smoke tests
+      runs: |
+        promu --help
+        promu help crossbuild 2>&1 | grep 'Crossbuild a Go project'
+        promu help codesign 2>&1 | grep 'Code sign the darwin binary'
+        promu help release 2>&1 | grep 'Upload all release files'
+    - uses: git-checkout
+      with:
+        repository: https://github.com/prometheus/promu.git
+        tag: v${{package.version}}
+        expected-commit: 3912dec4ab83971903015cc7b2a8d8ff93b73910
```

Comment:
this mostly causes build failures during package updates because wolfi-bot changes the expected commit in the package section not here 🫠